src/main/java/com/minecolonies/coremod/colony/buildings/AbstractBuildingCrafter.java

```diff
@@ -99,20 +99,21 @@ public boolean canBeGathered()
                     {
                         for (final ItemStorage itemStorage : recipeStorage.getCleanedInput())
                         {
-                            int amount = itemStorage.getAmount();
+                            int amount = itemStorage.getAmount() * request.getRequest().getCount();
```

Comment:
You will probably need to change that here then too if you adapt the AbstractBuilding to use the Delta in the request.